swap-default.conf

```diff
@@ -0,0 +1,58 @@
+################################################################################
+# Defaults are optimized for general usage
+################################################################################
+
+################################################################################
+# You can override any settings with files in:
+# /etc/systemd/swap.conf.d/*.conf
+################################################################################
+
+################################################################################
+# Zswap
+#
+# Kernel >= 3.11
+# Zswap create compress cache between swap and memory to reduce IO
+# https://www.kernel.org/doc/Documentation/vm/zswap.txt
+
+zswap_enabled=1
+zswap_compressor=zstd     # lzo lz4 zstd lzo-rle lz4hc
+zswap_max_pool_percent=25 # 1-99
+zswap_zpool=z3fold        # zbud z3fold (note z3fold requires kernel 4.8+)
+
+################################################################################
+# ZRam
+#
+# Kernel >= 3.15
+# Zram compression streams count for additional information see:
+# https://www.kernel.org/doc/Documentation/blockdev/zram.txt
+
+zram_enabled=0
+zram_size=$(( RAM_SIZE / 4 ))    # This is 1/4 of ram size by default.
+zram_count=${NCPU}               # Device count
+zram_streams=${NCPU}             # Compress streams
+zram_alg=zstd                    # See $zswap_compressor
+zram_prio=32767                  # 1 - 32767
+
+################################################################################
+# Swap File Chunked
+# Allocate swap files dynamically
+# For btrfs fallback to swapfile + loop will be used
+# ex. Min swap size 512M, Max 8*512M
```

Comment:
Should say 256M and 32*256M (this is actually my mistake, but I'll put it on you to fix it hehe)